src/test/scala/thurloe/crypto/EncryptionSpec.scala

```diff
@@ -0,0 +1,29 @@
+package thurloe.crypto
+
+import com.typesafe.config.ConfigFactory
+import org.scalatest.FunSpec
+
+import scala.util.{Failure, Success}
+
+class EncryptionSpec extends FunSpec{
+
+  describe("The Encryption") {
+    it("should encrypt and decrypt stuff") {
+      val enc = Aes256Cbc
+
+      val cryptoConfig = ConfigFactory.load().getConfig("crypto")
+      val secretKey = SecretKey(cryptoConfig.getString("key"))
+
+      val plaintextString = "0123456789ABCDEF, etc"
+      val plaintextBytes = plaintextString.getBytes("UTF-8")
+
+      val encrypted = enc.encrypt(plaintextBytes, secretKey)
+      val decrypted = encrypted flatMap { x => enc.decrypt(x, secretKey) map { new String(_, "UTF-8") } }
+
```

Comment:
would be good to also add a test that the encrypted value is either (a) what we expect OR (b) not the plaintext value.  Currently the test would still pass if we replaced the encryption with a no-op and it would be good to catch that.